docs/metadata.rst

```diff
@@ -181,14 +181,15 @@ Usage for Automatic Interface Generation and NatSpec
 ====================================================
 
 The metadata is used in the following way: A component that wants to interact
-with a contract (e.g. Mist or any wallet) retrieves the code of the contract,
-from that the IPFS/Swarm hash of a file which is then retrieved.  That file
+with a contract (e.g. Mist or any wallet) retrieves the code of the contract.
```

Comment:
Maybe we should not mention Mist anymore?

Reply:
yea I was also thinking the same